README.md

```diff
@@ -1,13 +1,11 @@
-# TCC-2020
+# Application of Natural Language Processing to detect “Turning Points” in Chess matches
 
-## Natural Language Chess
+This repository contains the code implementation for the undergraduate thesis on the machine learning area carried out Douglas Ramos, Pedro Felipe and Rafael Higa under the guidance of Professor Glauber, from University of São Paulo 2020.
```

Comment:
there's a missing "by" after "carried out".
Other than that, LGTM

Reply:
done :)